scripts/edd-get-appdata

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python
```

Comment:
Python convention is to have all code in packages / modules so we can test it, and then use entry_points. Not a fan of this commit.